tests/common.h

```diff
@@ -28,6 +28,9 @@
 #include "event.h"
 #include "group.h"
 
+#include <seasidecache.h>
```

Comment:
is this an entirely new dependency, needing .spec addition?  I assume not, but thought I'd check